Source: written by Volodymyr Medvid <vmedvid@riseup.net>
Upstream: no
Reason: Use system-provided CLucene, QuaZip, Crypto++ and SQLite

 lib/CMakeLists.txt |  5 ++---
 src/CMakeLists.txt | 13 +++++--------
 2 files changed, 7 insertions(+), 11 deletions(-)

diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index 8a9cbb9..e482a33 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -8,9 +8,8 @@ if(NOT ZLIB_FOUND)
     set(ZLIB_LIBRARY zlib)
 endif(NOT ZLIB_FOUND)
 
-add_subdirectory(cryptopp)
-add_subdirectory(clucene)
-add_subdirectory(quazip)
+find_package(CLucene)
+find_package(QuaZip)
 add_subdirectory(aggregation)
 add_subdirectory(extensionsystem)
 add_subdirectory(rapidjson)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c8c8758..a78d2a9 100755
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -27,7 +27,6 @@ set(wiznote_SOURCES
     share/wizzip.cpp
     share/wizqthelper.cpp
     share/wizmisc.cpp
-    share/sqlite3.c
     share/cppsqlite3.cpp
     share/wizobject.cpp
     share/wizkmcore.cpp
@@ -158,7 +157,6 @@ set(wiznote_HEADERS
     share/wizqthelper.h
     share/wizmisc.h
     share/sqlite3ext.h
-    share/sqlite3.h
     share/cppsqlite3.h
     share/wizobject.h
     share/wizkmcore.h
@@ -414,9 +412,7 @@ include_directories(
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${CMAKE_CURRENT_BINARY_DIR}
     ${CMAKE_SOURCE_DIR}/lib
-    ${CMAKE_SOURCE_DIR}/lib/clucene/src/core
-    ${CMAKE_SOURCE_DIR}/lib/clucene/src/shared
-    ${CMAKE_BINARY_DIR}/lib/clucene/src/shared
+    ${CMAKE_PREFIX_PATH}
 )
 
 link_directories(
@@ -424,10 +420,11 @@ link_directories(
 )
 
 target_link_libraries(${PROJECT_NAME}
+    sqlite3
     quazip
-    cryptlib
-    clucene-core-static
-    clucene-shared-static
+    cryptopp
+    clucene-core
+    clucene-shared
     extensionsystem
     aggregation
     coreplugin
-- 
2.2.2

